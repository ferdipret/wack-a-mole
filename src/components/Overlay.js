import styled from "styled-components"


export default styled.div`
  align-items: center;
  background-color: hsla(0, 0%, 0%, 0.8);
  color: white;
  display: flex;
  font-size: 20px;
  font-weight: bold;
  height: 100%;
  justify-content: center;
  position: absolute;
  text-align: center;
  width: 100%;
  z-index: 1;
`
