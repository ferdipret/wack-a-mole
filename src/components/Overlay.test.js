import React from "react"
import { shallow } from "enzyme"
import Overlay from "./Overlay"

describe("Overlay", () => {
  const wrapper = shallow(<Overlay />)
  it("renders properly", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
