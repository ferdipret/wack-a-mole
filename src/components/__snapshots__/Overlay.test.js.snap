// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay renders properly 1`] = `
<div
  className="sc-bdVaJa dkzApV"
/>
`;
